api-reference/beta/resources/user.md

```diff
@@ -78,10 +78,13 @@ by providing a [delta](../api/user_delta.md) function.
 |mobilePhone|String|The primary cellular telephone number for the user.|
 |mySite|String|The URL for the user's personal site.|
 |officeLocation|String|The office location in the user's place of business.|
+|onPremisesDomainName|String| Contains the on-premises DNS domain name synchronized from the on-premises directory. The property is only populated for customers who are synchronizing their on-premises directory to Azure Active Directory via Azure AD Connect. Read-only. |
```

Comment:
Instead of "DNS domain name" we should say "domainFQDN, also called dnsDomainName". This matches the Azure AD Connect documentation: https://docs.microsoft.com/en-us/azure/active-directory/connect/active-directory-aadconnectsync-attributes-synchronized